web/src/main/webapp/libs/app.bundle.min.js

```diff
@@ -2957,6 +2962,8 @@
         })
     }
     n(436), n(437), n(438), n(439), n(440), n(441), n(442), n(443), n(444), n(445), n(446), n(447), n(448), n(449), n(450), n(451), n(452);
+    n(502), n(503), n(504), n(505), n(506), n(507), n(508), n(509), n(510), n(511), n(512), n(513), n(514), n(515), n(516), n(517), n(518), n(519), n(520), n(521), n(522), n(523), n(524), n(525), n(526), n(527), n(528), n(529), n(530), n(531), n(532), n(533), n(534), n(535), n(536), n(537), n(538), n(539), n(540), n(541), n(542), n(543), n(544), n(545), n(546), n(547), n(548), n(549), n(550), n(551), n(552), n(553), n(554), n(555), n(556), n(557), n(558), n(559), n(560), n(561), n(562), n(563), n(564), n(565), n(566), n(567), n(568), n(569), n(570), n(571), n(572), n(573), n(574), n(575), n(576), n(577), n(578), n(579), n(580), n(581), n(582), n(583), n(584), n(585), n(586), n(587), n(588), n(589), n(590), n(591), n(592), n(593), n(594), n(595), n(596), n(597), n(598), n(599), n(600), n(601), n(602); // BAO
+
```

Comment:
For date localization.
Call [ e.defineLocale("af")] on line 73091 to [ e.defineLocale("zh-tw")] on line 7984.
By the way, the localization of dates is only supported in some languages, which is a bug in Jitsi Meet.